dev-python/apsw/apsw-3.35.4_p1.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..10} )
+
+inherit distutils-r1 toolchain-funcs
+
+MY_PV=${PV/_p/-r}
+MY_P=${PN}-${MY_PV}
+
+DESCRIPTION="APSW - Another Python SQLite Wrapper"
+HOMEPAGE="https://github.com/rogerbinns/apsw/"
+SRC_URI="https://github.com/rogerbinns/apsw/releases/download/${MY_PV}/${MY_P}.zip -> ${P}.zip"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="ZLIB"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc64 ~x86"
+IUSE="doc"
+
+RDEPEND=">=dev-db/sqlite-${PV%_p*}"
+DEPEND="${RDEPEND}"
+BDEPEND="app-arch/unzip"
+
+PATCHES=( "${FILESDIR}/${PN}-3.6.20.1-fix_tests.patch" )
+
+python_prepare_all() {
+	sed -e "s/gcc/$(tc-getCC)/" -i setup.py || die
```

Comment:
Could you maybe try sedding it into `os.environ.get("CC", "gcc")` maybe? That should be more reliable (i.e. not fail when CC happens to contain slashes ;-)), and worth submission upstream.